main.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"fmt"
+	"github.com/BuxOrg/go-buxclient/xpriv"
+	"os"
+)
+
+func main() {
+
+	keys, err := xpriv.Generate()
+
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	xpubKeyName := "xpub_key.txt"
+	xprvKeyName := "xprv_key.txt"
+
+	xpubKeyFile, err := os.Create(xpubKeyName)
+
+	if err != nil {
+		fmt.Println(err)
```

Comment:
It should rather stop the program if there is an error. So for example panic or os.Exit or log.Fatal will be good here.

Reply:
Done